app/views/layouts/application.html.erb

```diff
@@ -73,9 +73,9 @@
     <main class="govuk-main-wrapper" role="main">
       <div class="govuk-grid-row">
         <% if @page.navigation %>
-          <nav role="navigation" class="govuk-grid-column-one-third desktop-menu" aria-label="Desktop menu navigation">
+          <div class="govuk-grid-column-one-third desktop-menu" aria-label="Desktop menu">
             <%= render 'layouts/desktop_menu' %>
```

Comment:
Ignore what I wrote below - it was about the mobile_menu partial, not the desktop menu



I think I removed something like this last week because it was triggering an Axe error. It was about duplicate landmarks or something

Can you make sure you've merged main into this branch and that all the specs pass ?  I added a spec to test it - content-spec.rb